RELEASE.md

```diff
@@ -1,39 +1,39 @@
 # Release Process
-zfs-localpv follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/10).
+zfs-localpv follows a on quaterly release cadence for minor version releases. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/78).
 
-## Release Candidate Verification Checklist
+## Pre-release Candidate Verification Checklist
 
-Every release has release candidate builds that are created starting from the third week into the release. These release candidate builds help to freeze the scope and maintain the quality of the release. The release candidate builds will go through:
+Every release has a prerelease tag that gets created on branch creation, explained further below. This prerelease tag is meant for all the below action items throughout the release process:
 - Platform Verification
 - Regression and Feature Verification Automated tests.
-- Exploratory testing by QA engineers
-- Strict security scanners on the container images
-- Upgrade from previous releases
+- Exploratory testing by QA engineers.
+- Strict security scanners on the container images.
+- Upgrade from previous releases.
 - Beta testing by users on issues that they are interested in.
-- Dogfooding on OpenEBS workload and e2e infrastructure clusters.
 
-If any issues are found during the above stages, they are fixed and a new release candidate builds are generated.
+If any issues are found during the above stages, they are fixed and the prerelease tag is overidden by the newer changes and are up for above action items again.
```

Comment:
"and are up for above action items again" means?

Reply:
What I mean is, the tag is a fixed one and newer changes also are included in the same tag, for example 2.7.1-prerelease will have all the incremental changes going into release/2.7 branch till 2.7.1 is released.

Reply:
> and are up for above action items again" means?

Means everytime a new change goes in release/x.y from the point in time from creation of the branch release/x.y, the same prerelease tag is to be used for all testing and those list of items prior to release.

Reply:
What is this tag we're talking about here?

Reply:
It's the 2.7.0-prerlease and all, only the version, not actually a git tag.